x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/api/start.ts

```diff
@@ -82,7 +85,7 @@ export const registerSiemRuleMigrationsStartRoute = (
               const { exists, started } = await ruleMigrationsClient.task.start({
                 migrationId,
                 connectorId,
-                invocationConfig: { callbacks },
+                invocationConfig: { callbacks, configurable: { skipPrebuiltRulesMatching } },
```

Comment:
What is this `configurable` entry? I was not aware of it

Reply:
done `/evaluate` route here : https://github.com/elastic/kibana/pull/222542/commits/28c17ad22b07224aa224aa91051d4d6b45b9f63c